### PR TITLE
VEN-633 | Populate expandable customer details with values

### PR DESCRIPTION
**Pull request by santtul**
## Description :sparkles:
Show following data in customer details:
- comment
- berths
- boats
- applications

There will be another ticket that populates data in the table columns.

### Closes :no_good_woman:
https://helsinkisolutionoffice.atlassian.net/browse/VEN-633

### Automated tests :gear:️
- Customer list utils test
- Component render test `CustomerDetails.test.tsx`

### Manual testing :construction_worker_man:
- Go to customers
- Write "toivonen" to search box, you should get one hit
- Expand the row and you should see the values populated

## Screenshots :camera_flash:
ae267ca1-3c77-4bac-a5f7-4686cfbcdcc8

### CODE REVIEW

File: src/features/customerList/__tests__/utils.test.ts [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+      expect(application.createdAt).toEqual('2020-05-27T08:56:05.630976+00:00');
+
+      expect(profile.berthLeases.length).toEqual(2);
+      const berthLease = profile.berthLeases[0];
+      expect(berthLease.title).toEqual('Puotilan venesatama A 37');
+    });
+  });
+});
+
+const customersResponse: CUSTOMERS = {
```
Comment (TatuArvela) [inline src/features/customerList/__tests__/utils.test.ts:36]: This should maybe be extracted into a fixture for a cleaner test file

File: src/features/customerList/__tests__/utils.test.ts [hunk 2]
```diff
@@ -0,0 +1,153 @@
+import { getCustomersData } from '../utils';
+import { CUSTOMERS } from '../__generated__/CUSTOMERS';
+import { ContactMethod, ServiceType } from '../../../@types/__generated__/globalTypes';
+
+describe('utils', () => {
+  describe('getCustomersData', () => {
+    it('should convert data', () => {
```
Comment (TatuArvela) [inline src/features/customerList/__tests__/utils.test.ts:7]: This test could perhaps be several smaller tests, to have a better insight to the reason if/once it starts failing

File: src/features/customerList/__tests__/utils.test.ts (hunk 1 above)
Comment (santtul) [reply src/features/customerList/__tests__/utils.test.ts:36]: That is a solid idea. Will move to `__mocks__`

File: src/features/customerList/__tests__/utils.test.ts (hunk 2 above)
Comment (santtul) [reply src/features/customerList/__tests__/utils.test.ts:7]: Will split it.

**Pull request merged**